.github/process_pr.py

```diff
@@ -0,0 +1,55 @@
+import json
+import sys
+from typing import Any, Dict, Set, Tuple
+
+import requests
+
+# If the PR has any of these labels, we mark it as properly labeled.
+REQUIRED_LABELS = {
+    "TODO",
```

Comment:
Any suggestions what should go in here?

Reply:
Thank @pmeier !

We'll clean them as we go but for now I can think of:

- all the "module" labels
- all the "topic" labels
- enhancement
- bug
- code quality
- deprecation
- windows


CC @fmassa @datumbox 

Also, I thought we had a "New feature" tag but I don't see it? Did someone remove it?

Reply:
+1 on having the `module`/`topic` labels as a "top level" label. If we do adopt this, we need to change the way our release script works. At least the one we used for the upcoming release notes, classified as "unlabelled" PRs that had module and topic labels.

Not sure about `windows`, but no strong opinions.

The `new feature` makes sense to me, though I would like us to review all labels and provide concrete definitions to be aligned on those PRs that are on the boarder of two categories.

Reply:
@datumbox and I just went over our list of tags and we concluded that we could be using the following set  here:
```py
REQUIRED_LABELS = {
    'new feature',
    'bug',
    'code quality',
    'enhancement',
    'bc-breaking',
    'dependency issue',
    'deprecation',
    'module: c++ frontend',
    'module: ci',
    'module: datasets',
    'module: documentation',
    'module: io',
    'module: models.quantization',
    'module: models',
    'module: onnx',
    'module: ops',
    'module: reference scripts',
    'module: rocm',
    'module: tests',
    'module: transforms',
    'module: utils',
    'module: video',
    'Perf',
    'Revert(ed)',
}
```